service/worker/workerdeployment/client.go

```diff
@@ -187,23 +169,18 @@ func (d *ClientImpl) RegisterTaskQueueWorker(
 func (d *ClientImpl) DescribeVersion(
 	ctx context.Context,
 	namespaceEntry *namespace.Namespace,
-	seriesName string,
-	buildID string,
-) (_ *deploymentpb.DeploymentInfo, retErr error) {
+	version string,
```

Comment:
For now lets assume that version is globally unique, and we can always add the deploymentName parameter later if that changes